### PR TITLE
Use warn instead of Log4perl logger to output error message 

### PR DESCRIPTION
**Pull request by sasha-murzin**

### CODE REVIEW

**Comment by basiliscos** [comment]
new version, changes , upload to cpan, etc. :)

File: lib/App/Base.pm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package App::Base;
 use strict;
 use warnings;
-our $VERSION = "0.04";
+our $VERSION = "0.05";
```
Comment (basiliscos) [inline lib/App/Base.pm]: @sasha-murzin AFAIK, `$VERSION` should be specified once per distribution, not need to keep it for every file. Please, remove `$VERSION` from every place, except `App::Base`.

**Comment by basiliscos** [comment]
Fine for me

**Comment by shuwnyuantee** [comment]
looks good to me 👍

**Pull request merged**